habitat/plan.sh

```diff
@@ -33,17 +32,18 @@ pkg_deps=(
   core/libarchive
 )
 
-pkg_svc_user=root
+pkg_version() {
+  cat "$SRC_PATH/VERSION"
+}
 
 do_before() {
   do_default_before
   update_pkg_version
 }
 
 do_download() {
-  # Instead of downloading, build a gem based on the source in src/
-  cd $PLAN_CONTEXT/..
-  gem build $pkg_name.gemspec
+  build_line "Building gem from source. (${SRC_PATH}/${pkg_name}.gemspec)"
+  gem build "${SRC_PATH}/${pkg_name}.gemspec"
```

Comment:
To confirm, `${SRC_PATH}` is the directory the studio was entered from, right? Which should be fine for standard builds of this code base. Noting that this could surprise someone in their development environment if they enter a studio from higher up in a directory structure if they are trying to work with multiple plans in a single studio.

Reply:
you can't enter from higher up, you have to enter from the source root path so that the source is available inside the studio (docker container)